main.js

```diff
@@ -0,0 +1,48 @@
+
+$("button").on("click", function () {
+    let address = $("#addr-input").val()
+    let minPrice = $("#min-p-input").val()
+    let maxPrice = $("#max-p-input").val()
+    let minRooms = $("#min-r-input").val()
+    let maxRooms = $("#max-r-input").val()
+    let immediate = $("#immed-y")
+
+    let relevantApts = findRelevantApts(address, minPrice, maxPrice, minRooms, maxRooms, immediate)
```

Comment:
would look better if you sent it in an object

`const appartment = {address: '', minPrice: '', ...}`
`const relevantApts = find(RelevantApts(appartment)`